src/main.cpp

```diff
@@ -189,6 +189,8 @@ void runAgent ()
     mask_behavior.setVal(1);
 
     AgentContainer pc(geom, dm, ba, params.num_diseases, params.disease_names, params.fast);
+    bool stable_redistribute = !params.fast;
```

Comment:
This looks good. Just a note: params.fast only matters when the n^2 interactions models are used, because it only affects changes the binning.

Reply:
It also seems to be used in initialization if you are using the census option.

Reply:
Ah, yes, there is binning there too.